__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
```

Comment:
You're destructuring these only to put them back into an object below. Will be shorter to not bother:

```js
const testData = require('../db/data/test-data')
```